README.md

```diff
@@ -18,70 +19,75 @@ so using the Crossplane CLI in a Kubernetes cluster where Crossplane is
 ```
-make local-dev
+
+Then simply create a
+[`ProviderConfig`](./examples/provider-config/provider-config-incluster.yaml)
+that uses an `InjectedIdentity` source:
+  
+```console 
+kubectl apply -f ./examples/provider-config/provider-config-incluster.yaml
 ```
 
-Run controller against the cluster:
+`provider-helm` will then be installed and ready to use within the cluster. You
+can now create `Release` resources, such as [sample
+release.yaml](examples/sample/release.yaml).
 
-```
-make run
+```console
+kubectl create -f examples/sample/release.yaml
 ```
 
-Since controller is running outside of the Kind cluster, you need to make api server accessible (on a separate terminal):
+## Design 
 
-```
-sudo kubectl proxy --port=8081
-```
+See [the design
+document](https://github.com/crossplane/crossplane/blob/master/design/one-pager-helm-provider.md).
 
-### Testing in Local Cluster
+## Developing locally
 
-1. Prepare provider config for local cluster:
-    1. If helm provider running in cluster (e.g. provider installed with crossplane):
-    
-        ```
-        SA=$(kubectl -n crossplane-system get sa -o name | grep provider-helm | sed -e 's|serviceaccount\/|crossplane-system:|g')
-        kubectl create clusterrolebinding provider-helm-admin-binding --clusterrole cluster-admin --serviceaccount="${SA}"
-        kubectl apply -f examples/provider-config/provider-config-incluster.yaml
-        ```
-    1. If provider helm running outside of the cluster (e.g. running locally with `make run`)
-    
-        ```
-        KUBECONFIG=$(kind get kubeconfig --name local-dev | sed -e 's|server:\s*.*$|server: http://localhost:8081|g')
-        kubectl -n crossplane-system create secret generic cluster-config --from-literal=kubeconfig="${KUBECONFIG}" 
-        kubectl apply -f examples/provider-config/provider-config-with-secret.yaml
-        ```
+**Pre-requisite:** A Kubernetes cluster with Crossplane installed
 
-1. Now you can create `Release` resources with provider reference, see [sample release.yaml](examples/sample/release.yaml).
+To run the `provider-helm` controller against your existing local cluster,
+simply run:
 
-    ```
-    kubectl create -f examples/sample/release.yaml
-    ```
+```console
+make run
+```
 
-### Cleanup
+Since the controller is running outside of the local cluster, you need to make
+the API server accessible (on a separate terminal):
 
+```console
+sudo kubectl proxy --port=8081
 ```
-make local.down
+
+Then we must prepare a `ProviderConfig` for the local cluster (assuming you are
+using `kind` for local development):
+
+```console
+KUBECONFIG=$(kind get kubeconfig | sed -e 's|server:\s*.*$|server: http://localhost:8081|g')
+kubectl -n crossplane-system create secret generic cluster-config --from-literal=kubeconfig="${KUBECONFIG}" 
+kubectl apply -f examples/provider-config/provider-config-with-secret.yaml
 ```
+
+Now you can create `Release` resources with this `ProviderConfig`, for example
```

Comment:
wdyt about making it the same as above, something you can just copy:
```suggestion
Now you can create `Release` resources with this `ProviderConfig`, for example:
\```console
kubectl create -f examples/sample/release.yaml
\```
```

Reply:
good idea!! i added that easy copy/paste command again, but also left the link to the manifest so people can easily click that and see what it is while still in the browser